diff --git a/web/src/lib/api/api.ts b/web/src/lib/api/api.ts
index 0467adf3..91e43c20 100644
--- a/web/src/lib/api/api.ts
+++ b/web/src/lib/api/api.ts
@@ -98,6 +98,7 @@ const request = async (requestBody: CobaltSaveRequestBody, justRetried = false)
         redirect: "manual",
         signal: AbortSignal.timeout(20000),
         body: JSON.stringify(requestBody),
+        credentials: "include",
         headers: {
             "Accept": "application/json",
             "Content-Type": "application/json",
diff --git a/web/src/lib/api/server-info.ts b/web/src/lib/api/server-info.ts
index 0b905fc3..8ebef836 100644
--- a/web/src/lib/api/server-info.ts
+++ b/web/src/lib/api/server-info.ts
@@ -16,6 +16,7 @@ const request = async () => {
 
     const response: CobaltServerInfoResponse = await fetch(apiEndpoint, {
         redirect: "manual",
+        credentials: "include",
         signal: AbortSignal.timeout(10000),
     })
     .then(r => r.json())
diff --git a/web/src/lib/api/session.ts b/web/src/lib/api/session.ts
index 40304672..242f8e1c 100644
--- a/web/src/lib/api/session.ts
+++ b/web/src/lib/api/session.ts
@@ -21,6 +21,7 @@ export const requestSession = async () => {
         method: "POST",
         redirect: "manual",
         signal: AbortSignal.timeout(10000),
+        credentials: "include",
         headers: requestHeaders,
     })
     .then(r => r.json())
